### ISSUE TITLE
question about the training steps.

### ISSUE THREAD

**Issue by Clustering1** [CHELSEA234/M2F2_Det#13]
I attempted to use the binary classification detection model M2F2 with the bridge_adapter structure (instead of the binary classification detection model in llava), but after training, the loss has dropped to convergence, and the prediction results output by llava have relatively poor metrics. I found that the results given by llava are basically all fake. My training steps were carried out in accordance with the steps in the readme.

Also, I have the following questions about the pre-training weights you provided in the readme: “Download FF++ (test only) and moify this line. Then put M2F2-Det's detector-only weights [Google Drive]  in checkpoints/stage_1 and run…" (named current_model_180.pth), and "We offer an available template and required pre-trained weights in Google Drive." (named M2F2_Det_densenet121.pth) What is the difference between these two weights? I think the current_model_180.pth obtained after the first stage of training is the deepfake_model_path in the config.json of the subsequent llava. Is this correct?
